draft-ietf-capport-architecture.xml

```diff
@@ -254,8 +254,13 @@
         supported.
         </t>
         <t>
-        If User Equipment supports the Captive Portal API, it MUST validate the API server's TLS certificate
-        (see <xref target="RFC2818"/>). An Enforcement Device SHOULD allow access to
+        If User Equipment supports the Captive Portal API, it MUST validate the API server's
+        TLS certificate (see <xref target="RFC2818"/>) according to the procedures in
+        <xref target="RFC6125"/>.  The API server's URI is obtained via a network
+        provisioning protocol, which will typically provide a hostname to be used
+        in TLS server certificate validation, against a DNS-ID in the server certificate.
+        If the API server is identified by IP address, the iPAddress subjectAltName
+        is used to validate the server certificate.  An Enforcement Device SHOULD allow access to
```

Comment:
Looking at RFC6125 (section B.2), maybe should replace "the iPAddress subjectAltName is used to validate the server certificate." with, "... the iPAddress subjectAltName must be present in the certificate and must exactly match the IP address in the API server's URI."

Reply:
I don't think that we need to copy requirements.  The citation exists.  This is merely assistance in finding the right pieces in 6125.

Reply:
Thanks for the input. I'll keep Ben's phrasing for now.